internal/engine/eval/templates/trustyTemplate.tmpl

```diff
@@ -0,0 +1,17 @@
+{{- if .maliciousPackages -}}
+Malicious packages:
+{{- range .maliciousPackages }}
+* {{ . }}
+{{- end }}
+{{- end }}
+
+{{- /* Add a newline only if both maliciousPackages and lowScoringPackages exist */ -}}
+{{- if and .maliciousPackages .lowScoringPackages}}
+{{ end -}}
+
```

Comment:
I believe lines 7 and 11 already add newlines, what's the difference if we remove this `if`?

Reply:
If we remove this `if` and there are only low scoring packages, and no malicious packages, then the message has a leading `\n`: `"\nPackages with a low Trusty score:\n* package"`. It's not a big problem, but it was bugging me.